### ISSUE TITLE
 SD Manual - Rentention Settings - Clarification #262 

### ISSUE THREAD

**Issue by rgleason** [Rasbats/managed_plugins#15]
See [Manual Clarification about Retention Settings](https://github.com/Rasbats/shipdriver_pi/issues/232#issuecomment-950839166)

**Comment by leamas** [comment]
We need to be careful here: this is not really about the shipdriver workflow, this is about the cloudsmith organization. As I see it, it applies to both shipdriver and testplugin, but OTOH not to all users.

In short: this is not about shipdriver. It needs a separate heading, probably already in the Plugins Manual, which both shipdriver and testplugin could reference (i. e., link to). It would of course contain a disclaimer to which users it applies.

**Comment by rgleason** [comment]
Yes, agreed. But lets leave it here for now. So I know where it is.
